testutil/ciao-down/cloudinit.go

```diff
@@ -94,7 +94,7 @@ runcmd:
  - echo "PATH=\"$PATH:/usr/local/go/bin:{{$.GoPath}}/bin:/usr/local/nodejs/bin\""  >> /etc/environment
 
  - curl -X PUT -d "Downloading Go" 10.0.2.2:{{.HTTPServerPort}}
- - {{template "ENV" .}}wget https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz -O /tmp/go1.8.linux-amd64.tar.gz
```

Comment:
Not really related to what you're changing in this patch, but:

If this were split into an url and file in a const struct like with guestDownloadURL, there wouldn't be duplication of the strings for the filename (eg: "go1.8.linux-amd64.tar.gz").

It would be better to not do this in something world writeable like /tmp, versus ~/local.

Is there a reason to use both curl and wget here?  I suggest removing wget and simply using curl consistently, instead of propagating the wget usage over into testutil/ciao-down/prepare.go downloadFN().